modules/python/python-fetcher.c

```diff
@@ -566,6 +568,10 @@ py_log_fetcher_init(void)
                        PyLong_FromLong(THREADED_FETCH_NOT_CONNECTED));
   PyDict_SetItemString(py_log_fetcher_type.tp_dict, "FETCH_SUCCESS",
                        PyLong_FromLong(THREADED_FETCH_SUCCESS));
+  PyDict_SetItemString(py_log_fetcher_type.tp_dict, "TRY_AGAIN",
```

Comment:
Other constants have the `FETCH_` prefix, it should be consistent.

Reply:
Fixed.